docs/quantum_keycodes.md

```diff
@@ -8,8 +8,9 @@ On this page we have documented keycodes between `0x00FF` and `0xFFFF` which are
 
 ## QMK Keycodes :id=qmk-keycodes
 
-|Key           |Aliases  |Description                                            |
-|--------------|---------|-------------------------------------------------------|
-|`RESET`       |         |Put the keyboard into bootloader mode for flashing     |
-|`DEBUG`       |         |Toggle debug mode                                      |
-|`EEPROM_RESET`|`EEP_RST`|Reinitializes the keyboard's EEPROM (persistent memory)|
+|Key           |Aliases  |Description                                                                      |
+|--------------|---------|---------------------------------------------------------------------------------|
+|`RESET`       |         |Put the keyboard into bootloader mode for flashing                               |
+|`DEBUG`       |         |Toggle debug mode                                                                |
+|`EEPROM_RESET`|`EEP_RST`|Reinitializes the keyboard's EEPROM (persistent memory)                          |
+|`KC_MAKE`     |         |Sends `qmk compile -kb (keyboard) -km (keymap)`, or `qmk flash` if shift is held |
```

Comment:
IMHO this should use the `QK_` prefix instead, I think `KC_` should be reserved for basic keycodes.